lib/install/management/resource_pool.go

```diff
@@ -93,6 +93,57 @@ func setResources(spec *types.ResourceAllocationInfo, resource types.ResourceAll
 	}
 }
 
+func (d *Dispatcher) destroyVMGroup(conf *config.VirtualContainerHostConfigSpec) error {
```

Comment:
trivial question: why do we want this function to be in resource_pool.go? since function `createVMGroup` is in install/management/create.go, does it make more sense if we have `destroyVMGroup` in install/management/delete.go

Reply:
Both functions should probably be in a new `vm_groups.go` (eventually, we'll want to re-use them for configure). Good catch!